solana/pyth2wormhole/program/src/migrate.rs

```diff
@@ -64,6 +59,18 @@ pub fn migrate(ctx: &ExecutionContext, accs: &mut Migrate, data: ()) -> SoliResu
         ));
     }
 
+    if *accs.system_program.key != system_program::id() {
```

Comment:
Where is it used? by `create` method below?

Reply:
correct, we were using a solitaire-client thingy for this previously, which would include the system program automatically for accounts that will possibly need it for creation. Latest solitaire is on-chain focused.